### ISSUE TITLE
Critical error in 3.3LTS export every time, as long as any material uses image texture (same file works fine in 3.1) - "cannot pickle Image object"

### ISSUE THREAD

**Issue by romandesign** [AsoboStudio/glTF-Blender-IO-MSFS#222]
### Current Behavior

Every time I try to export with Blender 3.3LTS I get an error. Same files that work perfectly with Blender 3.1 always throw an error.
No matter what I do this error always happens. I tried different Blender versions, disabling other blender plugins etc. - but it's always the same. Please advise! It should be working with 3.3.5LTS but it doesn't. This happens even with a default cube with the most basic material that has at least one image. If no material refers to any image, there is no error, but as soon as image  texture is added to any material, there's this "cannot pickle Image object" error. I tried Blender 3.3.0LTS - same error. I also updated Python to the latest version - no change either. I tried manual addon install and install via settings. I tried latest release and a direct code from the latest changes - always the same. I don't know what else to try and what can be worn with my environment, as I don't see others complain about this critical issue.
![image](https://user-images.githubusercontent.com/9002820/229305877-cb40b001-3459-4360-9212-18cb76bcaddd.png)

### Expected Behavior

No error should be throws, file exports fine in Blender 3.1 and I tried it on many files that all behave the same way.

### Steps To Reproduce

Take any file, set up a material with any image texture, set up for exporting the object and click export.

### Environment

```markdown
- OS: Win11 (also tried in Win10 on a different partition - same result)
- Blender: 3.3.5LTS
- glTF-Blender-IO-MSFS: latest version 1.3.0
```

**Comment by romandesign** [comment]
Update: it seems to only affect files that were originally created i Blender 3.1, or derived from them. Even if I take a working 3.1 file, delete all materials, delete all objects, and then create a new cube and pull new material from library for it - I get an error. but if I create a new file from scratch and do the cube and pull the material - I have no error. As all my objects were naturally created in previous Blender versions, it's not acceptable to not be able to open them and export again. There must be a way to make those files compatible. Is there anything I can do for the old files to work?

**Comment by romandesign** [comment]
I found the culprit! All I have to do to make the error disappear is uncheck "custom properties" in export settings of the addon. It's still a bug. I'm not sure if custom properties are really needed, but if the option is there it should not cause error like that.

**Comment by lonesparrow67** [comment]
I am having this exact same issue with 3.3.5 and exporter 1.3.0, and it isn't isolated to just a model made in a prior Blender version...I created a brand new cube mesh in 3.3.5 and when going to export it, got the same exact pickle error....

Unchecking the Custom Properties option did indeed work, thank you for that! 

A better question is, and excuse my ignorance, what exactly does that custom properties option do? Does MSFS even recognize it? What are we losing/missing by unchecking that, if anything?

Thanks!

**Comment by romandesign** [comment]
I don't really know if Custom properties do anything. No anything I know of. Regardless, it should either be supported, ot not generate the error, or the option should be taken out.

**Comment by y-khodja** [comment]
Hello,

Can you send us your Blend File to test it ?

Thank you !

**Comment by romandesign** [comment]
[CYYJ_Streetlamp.zip](https://github.com/AsoboStudio/glTF-Blender-IO-MSFS/files/11919381/CYYJ_Streetlamp.zip)

Here's the file. All my files are doing that. You can replace missing images with any PNGs. File packed with all images is too large for uploading here. I get this error in 3.3LTS with all file use all kind of materials, every time.

**Comment by romandesign** [comment]
@y-khodja in addition to the file I sent - I just downloaded the freshly released 3.1 and I can confirm that I still get the same error. In addition to that, there is a new bug: when I chose the directory for output in the plugin's interface after I select the new directory, all I get in the field is "//". It doesn't input the selected output directory as it used to anymore. I can only type it manually, but the normal Blender selection dialog doesn't work now.

**Comment by vmo-cchiasson** [comment]
I'm having the same issue (even went as far as to remake all my materials using Blender 3.3.3 and Asobo export tool 1.3.0 (listed as 1.3.1 in the readme). I'm at a loss for what is causing this image picking error and I hope they can fix it in a future release.

**Comment by y-khodja** [comment]
Hello !

I have looked at your scene, you should not export with custom propreties checked. The plugin adds to Blender Custom propreties that are used to set the correct extensions in the export. I will set it unchecked in default mode, it should be used only when you are not exporting a model with the Microsoft Flight Simulator Extensions enabled.

The "//" is the relative path of where your project is set (it's relative to were your scene is saved), it has been changed for design purpose.

I hope this will help you !

Thank you !

> @y-khodja in addition to the file I sent - I just downloaded the freshly released 3.1 and I can confirm that I still get the same error. In addition to that, there is a new bug: when I chose the directory for output in the plugin's interface after I select the new directory, all I get in the field is "//". It doesn't input the selected output directory as it used to anymore. I can only type it manually, but the normal Blender selection dialog doesn't work now.

**Comment by romandesign** [comment]
> I hope this will help you !

Thanks for the reply and clarification. I assumed from what I read that the "custom properties" were required for MSFS and some features won't work without them. But it seems that everything works without it, so it seems they are not required, as you say. As this is undocumented, I believe, and very confusing, I suggest making this checkbox mutually exclusive with MSFS export, or adding a comment to the interface that it should be disabled for MSFS, as it's not possible to figure out otherwise except by a lot of experimentation.
